pkgs/tools/networking/zerotierone/default.nix

```diff
@@ -20,9 +44,19 @@ stdenv.mkDerivation rec {
       --replace 'armv5' 'armv6'
   '';
 
-
-  nativeBuildInputs = [ ronn ];
-  buildInputs = [ openssl lzo zlib iproute2 ];
+  nativeBuildInputs = [
+    pkg-config
+    ronn
+    rustPlatform.cargoSetupHook
+    rustPlatform.rust.cargo
+    rustPlatform.rust.rustc
+  ];
+  buildInputs = [
+    iproute2
+    lzo
+    openssl
+    zlib
+  ];
 
   enableParallelBuilding = true;
 
```

Comment:
checkPhase and installPhase should have the pre* and post hooks called

Reply:
> Also unsure if the installPhase shouldn't just be postInstall.

I'm also unsure -- cleaning up after stock "make install" looks like more work than just doing our own

Reply:
... but it is indeed more correct :)